docs/source/dpo_trainer.mdx

```diff
@@ -0,0 +1,72 @@
+# DPO Trainer
```

Comment:
I think you need to add that file in `_toctree.yml` file to see it on the documentation

Reply:
ah yes i fixed a conflict so will add it back